### ISSUE TITLE
Invite Icon is Wrong

### ISSUE THREAD

**Issue by lrojas94** [sharetribe/sharetribe#2419]
##### Sharetribe Version:

5.8

##### Environment:

Ubuntu

##### Server mode:

Prod.
##### Description:

The icon used to display "invite friends" is not displayed.

##### Steps To Reproduce:

Nothing, this will always happen on the site.

##### Additional information

The issue seems to be related to the css class used in there. According to the  repo's code:
https://github.com/sharetribe/sharetribe/blob/master/app/view_utils/icon_map.rb#L187

The class used for that should be **icon-group**, as it is presented in the old font-awesome docs.

**Comment by rap1ds** [comment]
Thanks for reporting the issue!

It's known that `font-awesome` icons are not up-to-date. There are a lot of icons in the `ss-pika` icon set that are not added to `font-awesome`.

We are happy to merge contributions!

**Comment by matheusvetor** [comment]
I want to work on that.

**Comment by matheusvetor** [comment]
we can consider using ```font-awesome-rails``` gem instead of using the fonts alone?

**Comment by rap1ds** [comment]
Hmm... maybe! I haven't though about that before (didn't know that such gem exists).

What are the benefits? No need to maintain the .css and font files in the repository?

**Comment by matheusvetor** [comment]
@rap1ds yes, no need to maintain css and fonts and it comes with ```fa_icon``` helper: 
```
fa_icon "camera-retro"
# => <i class="fa fa-camera-retro"></i>
```

**Comment by matheusvetor** [comment]
#2729

**Comment by matheusvetor** [comment]
My PR is not passing on CI. Someone can help me with this?

**Comment by DanielAmah** [comment]
I will like to work on this bug as well. @rap1ds
